packages/synthetix-main/cannonfile.toml

```diff
@@ -161,6 +166,7 @@ modified = ["contracts/modules/common", "contracts/modules/account"]
 depends = [
   "contract.OwnerModule",
   "contract.UpgradeModule",
+  "contract.FeatureFlagModule",
```

Comment:
do all of the systems really need feature flags? is it a common module?

Reply:
moved to core; only pool and markets, but i only foresee us using it for core stuff anyway.  👍🏽